pkg/machinery/resources/cluster/affiliate.go

```diff
@@ -82,6 +82,14 @@ type AffiliateSpec struct {
 	OperatingSystem string                `yaml:"operatingSystem" protobuf:"5"`
 	MachineType     machine.Type          `yaml:"machineType" protobuf:"6"`
 	KubeSpan        KubeSpanAffiliateSpec `yaml:"kubespan,omitempty" protobuf:"7"`
+	ControlPlane    *ControlPlane         `yaml:"controlPlane,omitempty" protobuf:"8"`
+}
+
+// ControlPlane describes ControlPlane data if any.
+//
+//gotagsrewrite:gen
+type ControlPlane struct {
+	Port int `yaml:"port" protobuf:"1"`
```

Comment:
same as with discovery data:

```suggestion
	APIServerPort int `yaml:"port" protobuf:"1"`
```